devstack_vm/bin/check_neutron.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Make sure you add the loop part here. This script should check as we do it now in the CI and return the final result.